django_lightweight_queue/management/commands/queue_runner.py

```diff
@@ -22,13 +22,35 @@ class Command(NoArgsCommand):
             help="Total number of machines running the queues"),
         optparse.make_option('--only-queue', action='store', default=None,
             help="Only run the given queue, useful for local debugging"),
+        optparse.make_option('--num-queue-workers', action='store', default=None,
+            help="The number of queue workers to run, only valid when running a specific queue"),
```

Comment:
I wondered about that, but I couldn't work out what would then happen if the argument wasn't present (i.e: would it be `None` or `0`). (The docs didn't appear to say).

I wasn't worried as there are a number of other options here which are treated similarly.